mojo/core/broker_castanets.h

```diff
@@ -89,6 +89,7 @@ class BrokerCastanets : public Channel::Delegate,
   void StartChannelOnIOThread();
 
   bool host_;
+  bool tcp_connection_ = false;
```

Comment:
It seems simpler to use BrokerCastanets only for TCP Socket without |tcp_connection_| flag.

Reply:
On the renderer side, BrokerCastanets has to cover both TCP and Unix-domain connection. The connection type check is needed.